packages/smooth_app/lib/pages/preferences_v2/roots/dev_mode_root.dart

```diff
@@ -0,0 +1,666 @@
+    final UserPreferences userPreferences = context.watch<UserPreferences>();
+
+    return <PreferenceCard>[
+      PreferenceCard(
+        title: appLocalizations.dev_mode_section_data,
+        tiles: <PreferenceTile>[
+          PreferenceTile(
+            title: appLocalizations.background_task_title,
+            subtitleText: appLocalizations.background_task_subtitle,
+            trailing: const BackgroundTaskBadge(
+              child: Icon(Icons.edit_notifications_outlined),
+            ),
+            onTap: () async => Navigator.push<void>(
+              context,
+              MaterialPageRoute<void>(
+                builder: (BuildContext context) => const OfflineTaskPage(),
+              ),
+            ),
+          ),
+          PreferenceTile(
+            title: appLocalizations.offline_data,
+            onTap: () => Navigator.push<void>(
+              context,
+              MaterialPageRoute<void>(
+                builder: (BuildContext context) => const OfflineDataPage(),
+              ),
+            ),
+          ),
+          PreferenceTile(
+            title: appLocalizations.dev_preferences_export_history_title,
+            subtitleText: appLocalizations.clipboard_barcode_copy,
+            onTap: () async {
+              final LocalDatabase localDatabase = context.read<LocalDatabase>();
+              final Map<String, dynamic> export =
+                  await DaoProductList(localDatabase).export(
+                ProductList.history(),
+              );
+              final List<Widget> children = <Widget>[];
+              for (final String barcode in export.keys) {
+                final bool? exists = export[barcode] as bool?;
+                children.add(
+                  ListTile(
+                    leading: Icon(exists == null
+                        ? Icons.error
+                        : exists
+                            ? Icons.check
+                            : Icons.help_outline),
+                    title: Text(barcode),
+                    subtitle: Text(exists == null
+                        ? appLocalizations
+                            .dev_preferences_export_history_progress_error
+                        : exists
+                            ? appLocalizations
+                                .dev_preferences_export_history_progress_found
+                            : appLocalizations
+                                .dev_preferences_export_history_progress_not_found),
+                  ),
+                );
+              }
+
+              if (!context.mounted) {
+                return;
+              }
+              await showDialog<void>(
+                context: context,
+                builder: (BuildContext context) => SmoothAlertDialog(
+                  title: appLocalizations
+                      .dev_preferences_export_history_dialog_title,
+                  body: SizedBox(
+                    height: 400,
+                    width: 300,
```

Comment:
?

Reply:
This is copy pasted from the old dev_mode_page